src/pages/yujinLee/MyComponent/Feed.js

```diff
@@ -0,0 +1,124 @@
+import { useState } from 'react';
+
+const Feed = () => {
+  const [comments, newComment] = useState([]);
+  const [input, setInput] = useState('');
+
+  const handleInputChange = e => {
+    setInput(e.target.value);
+  };
+
+  const commentUploadBtn = () => {
+    newComment(prevState => {
+      setInput('');
+      return [...prevState, input];
+    });
```

Comment:
state의 함수형 업데이트!

Reply:
state(`comments`)에 접근할 필요 없이 setState 함수 안에서 이전의 state에 접근할 수 있다!

Reply:
이전의 state를 토대로 setState하는 게 아니라면, 굳이 함수형 업데이트로 하지 않아도 됨!
```suggestion
    setComments([...comments, input]);
```